### ISSUE TITLE
Cannot create user Issue with `databricks_user` resource because it already exists

### ISSUE THREAD

**Issue by rohitnijhawan** [databricks/terraform-provider-databricks#1622]
### Configuration
```hcl
# Copy-paste your Terraform configuration here
resource "databricks_user" "data_lake" {
     user_name = "data_lake@acme.com"
}
```

### Expected Behavior
Ignore or re-create user from SCIM or any other API

### Actual Behavior
Error message issued
Error: **cannot create user: User with username data_lake@acme.com already exists.**

### Steps to Reproduce
- Run the terraform exporter with `-services=user`
- Use the output to run `terraform apply` on a target workspace
- Run `terraform apply` again on a target workspace

### Terraform and provider versions
Tested with 1.3.0

### Important Factoids
The `databricks_user` and `databricks_service_principal` resources have a force flag which probably should be set to true by default.

**Comment by nfx** [comment]
use the [force](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/user#force), @rohitnijhawan :)

**Comment by rohitnijhawan** [comment]
Yes I did say that force=true is the solution. The problem is the exporter output.
The output can not be used to `terraform apply` if it's going to fail. I am trying to get us as close to a set of output (.tf) files that require no editing to be applied to new workspaces.

**Comment by rohitnijhawan** [comment]
> use the [force](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/user#force), @rohitnijhawan :)

Breaks when the user is an admin
